storage/entities/transaction.js

```diff
@@ -180,96 +159,104 @@ class Transaction extends BaseEntity {
 	constructor(adapter, defaultFilters = {}) {
 		super(adapter, defaultFilters);
 
+		this.addField('rowId', 'string', {
+			fieldName: 'trs.rowId',
+		});
+
+		this.addField('transferData', 'string', {
+			fieldName: 'trs.transferData',
+		});
+
 		this.addField('id', 'string', {
 			filter: filterTypes.TEXT,
-			fieldName: 't_id',
+			fieldName: 'trs.id',
 		});
+
 		this.addField('blockId', 'string', {
 			filter: filterTypes.TEXT,
-			fieldName: 'b_id',
 		});
+
 		this.addField('blockHeight', 'string', {
 			filter: filterTypes.NUMBER,
-			fieldName: 'b_height',
+			fieldName: 'height',
 		});
+
 		this.addField('type', 'number', {
 			filter: filterTypes.NUMBER,
-			fieldName: 't_type',
 		});
+
 		this.addField('timestamp', 'number', {
 			filter: filterTypes.NUMBER,
-			fieldName: 't_timestamp',
+			fieldName: 'trs.timestamp',
 		});
+
 		this.addField(
 			'senderPublicKey',
 			'string',
 			{
 				filter: filterTypes.TEXT,
 				format: 'publicKey',
-				fieldName: 't_senderPublicKey',
 			},
-			stringToByteOnlyInsert
+			stringToByte
 		);
+
 		this.addField(
 			'recipientPublicKey',
 			'string',
 			{
 				filter: filterTypes.TEXT,
 				format: 'publicKey',
-				fieldName: 't_recipientPublicKey',
+				fieldName: 'm.publicKey',
 			},
-			stringToByteOnlyInsert
+			stringToByte
 		);
+
 		this.addField(
 			'requesterPublicKey',
 			'string',
 			{
 				filter: filterTypes.TEXT,
 				format: 'publicKey',
-				fieldName: 't_requesterPublicKey',
 			},
-			stringToByteOnlyInsert
+			stringToByte
 		);
+
 		this.addField('senderId', 'string', {
 			filter: filterTypes.TEXT,
-			fieldName: 't_senderId',
 		});
+
 		this.addField('recipientId', 'string', {
 			filter: filterTypes.TEXT,
-			fieldName: 't_recipientId',
 		});
+
 		this.addField('amount', 'string', {
 			filter: filterTypes.NUMBER,
-			fieldName: 't_amount',
 		});
+
 		this.addField('fee', 'string', {
 			filter: filterTypes.NUMBER,
-			fieldName: 't_fee',
 		});
-		this.addField(
-			'signature',
-			'string',
-			{ fieldName: 't_signature' },
-			stringToByte
-		);
-		this.addField(
-			'signSignature',
-			'string',
-			{ fieldName: 't_SignSignature' },
-			stringToByte
-		);
-		this.addField('signatures', 'string', { fieldName: 't_signatures' });
+
+		this.addField('signature', 'string', {}, stringToByte);
+
+		this.addField('signSignature', 'string', {}, stringToByte);
+
+		this.addField('signatures', 'string');
+
+		this.addField('asset', 'string');
```

Comment:
can we add support to do like `asset.dapp.name`? or maybe in next version?

Reply:
Out of scope of this PR imo.